src/screens/GroupInfo/elements/ItemLink.js

```diff
@@ -40,13 +40,13 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600],
     fontSize: 14,
     lineHeight: 16.94,
-    color: COLORS.white
+    color: COLORS.almostBlack
   },
   domain: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
     lineHeight: 18,
-    color: COLORS.blackgrey
+    color: COLORS.gray400
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.blackgrey` to `COLORS.gray400` in the domain style should be checked for consistency with other similar components and ensure it maintains good contrast for readability.
```diff
-    color: COLORS.blackgrey
+    color: COLORS.gray400
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->